### PR TITLE
SWP-447 : Merging and testing navigation

### PR DESCRIPTION
**Pull request by naomiaro**

### CODE REVIEW

File: scripts/apps/web-publisher/controllers/WebPublisherManagerController.js [hunk 1]
```diff
@@ -7,24 +7,151 @@
...
+    $scope.menu = undefined;
+    $scope.menuObj = {};
+
     /**
      * @ngdoc method
      * @name WebPublisherManagerController#changeTab
      * @param {String} newTabName - name of the new active tab
      * @description Sets the active tab name to the given value
      */
+
```
Comment (gbbr) [inline scripts/apps/web-publisher/controllers/WebPublisherManagerController.js]: This blank line is unnecessary

File: scripts/apps/web-publisher/controllers/WebPublisherManagerController.js [hunk 2]
```diff
@@ -7,24 +7,151 @@
  * @requires https://docs.angularjs.org/api/ng/type/$rootScope.Scope $scope
  * @description WebPublisherManagerController holds a set of functions used for web publisher
  */
+
```
Comment (gbbr) [inline scripts/apps/web-publisher/controllers/WebPublisherManagerController.js]: Why the blank line? Makes it look a bit like the documentation doesn't belong to what is below it imo. But I don't feel strongly about it.

File: scripts/apps/web-publisher/controllers/WebPublisherManagerController.js [hunk 3]
```diff
@@ -7,24 +7,151 @@
...
+     */
+    self.newMenu = () => {
+        $scope.menus.push({
+            label: undefined
+        });
+    };
+
+    /**
+     * @ngdoc method
+     * @name WebPublisherManagerController#saveMenu
```
Comment (Aca-jov) [inline scripts/apps/web-publisher/controllers/WebPublisherManagerController.js]: Wrong name of function.

File: scripts/apps/web-publisher/services/PubAPIFactory.js [hunk 4]
```diff
@@ -29,7 +31,23 @@ export function PubAPIFactory(config, $http, $q) {
          * @description Change the tenant we are using the api for
          */
         setTenant(tenant) {
+            //the default tenant may not be configured.
+            if (tenant === DEFAULT_TENANT) {
+                tenant = '';
```
Comment (naomiaro) [inline scripts/apps/web-publisher/services/PubAPIFactory.js]: I should fix this to not wipe out 'default', but to maintain same URL building

**Pull request closed without merging**